<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>run_simulation</class>
 <widget class="QWidget" name="run_simulation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>140</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Run Simulations...</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="button_run_local">
     <property name="text">
      <string>Run Local</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="button_run_cluster">
     <property name="text">
      <string>Run on Cluster</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="button_run_aws">
     <property name="text">
      <string>Run on AWS</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="button_cancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
